### PR TITLE
issue: 969414 Change log severity from DEBUG to ERROR while not enough

### PR DESCRIPTION
**Pull request by liranoz12**
buffers in the pool

Signed-off-by: Liran Oz <lirano@mellanox.com>

### CODE REVIEW

File: src/vma/dev/buffer_pool.cpp [hunk 1]
```diff
@@ -386,7 +386,7 @@ mem_buf_desc_t *buffer_pool::get_buffers(size_t count, uint32_t lkey)
 	__log_info_funcall("requested %lu, present %lu, created %lu", count, m_n_buffers, m_n_buffers_created);
 
 	if (m_n_buffers < count) {
-		static vlog_levels_t log_severity = VLOG_DEBUG; // DEBUG severity will be used only once - at the 1st time
+		static vlog_levels_t log_severity = VLOG_WARNING; // WARNING severity will be used only once - at the 1st time
 
 		VLOG_PRINTF_INFO(log_severity, "not enough buffers in the pool (requested: %lu, have: %lu, created: %lu isRx=%d isTx=%d)",
```
Comment (OphirMunk) [inline src/vma/dev/buffer_pool.cpp]: @liranoz12 Please add the word "fail" anywhere inside the log message

**Comment by mellanox-github** [comment]
Test FAILed.
See http://bgate.mellanox.com/jenkins/job/accl-libvma-pr/927/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/778/ for details (Mellanox internal link).

**Comment by liranoz12** [comment]
@OphirMunk

**Comment by mellanox-github** [comment]
Test FAILed.
See http://bgate.mellanox.com/jenkins/job/accl-libvma-pr/928/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/779/ for details (Mellanox internal link).

**Comment by igor-ivanov** [comment]
:thumbsup:

**Comment by mellanox-github** [comment]
Test FAILed.
See http://bgate.mellanox.com/jenkins/job/accl-libvma-pr/956/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/786/ for details (Mellanox internal link).

**Comment by OphirMunk** [comment]
:+1:

**Comment by OphirMunk** [comment]
bot:retest

**Comment by mellanox-github** [comment]
Test FAILed.
See http://bgate.mellanox.com/jenkins/job/accl-libvma-pr/959/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/788/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://bgate.mellanox.com/jenkins/job/accl-libvma-pr/975/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://bgate.mellanox.com/jenkins/job/accl-libvma-pr/983/ for details (Mellanox internal link).

**Comment by liranoz12** [comment]
@OphirMunk  dont forget the edit the commit message from WARNING to ERROR...

**Comment by liranoz12** [comment]
bot:retest

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/accl-libvma-pr/1011/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/805/ for details (Mellanox internal link).

**Comment by OphirMunk** [comment]
Merged #294

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/accl-libvma-pr/1034/ for details (Mellanox internal link).

**Pull request closed without merging**